src/components/ControlPanel.vue

```diff
@@ -1054,7 +1054,7 @@ export default {
       this.setName(n)
     },
     setName (n) {
-      n._name = n.name || 'NodeID_' + n.node_id
+      n._name = [n.loc, n.name].join('_') || 'NodeID_' + n.node_id
```

Comment:
What if node loc doesn't exists?

Reply:
if n.loc is empty, you get an _<name> see my screenshot which is _test

Reply:
@billiaz I would prefer more user friendy names (some users may have underscore in their node names so I suggest: `n.name (n.location)` if location doesn't exist just use `n.name`

Reply:
n.loc and n.location returns the same?

Reply:
@billiaz Maybe I explained it wrong, I mean this `n.name (n.location)` is the result I want. Translated in code it becomes: `n.name + (n.loc ? ' (' + n.loc + ')' : ''`